### ISSUE TITLE
direct 'dump frame' from dataio-pyshovel (Trac #890)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#884]
please implement a direct way to dump a frame from dataio-pyshovel to file

[this was implemented in dataio-shovel  by pressing 'w'-key]

right now the dump of a single frame into a file requires going through the interactive console ('i') and typing the usual dataio-interface, which is tedious.

alsoa  function to jump to a certain event by event_id would be nice!

Migrated from https://code.icecube.wisc.edu/ticket/890
```json
{
    "status": "closed", 
    "changetime": "2019-01-11T21:50:21", 
    "description": "please implement a direct way to dump a frame from dataio-pyshovel to file\n\n[this was implemented in dataio-shovel  by pressing 'w'-key]\n\nright now the dump of a single frame into a file requires going through the interactive console ('i') and typing the usual dataio-interface, which is tedious.\n\nalsoa  function to jump to a certain event by event_id would be nice!", 
    "reporter": "mzoll", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1547243421691151", 
    "component": "combo core", 
    "summary": "direct 'dump frame' from dataio-pyshovel", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2015-03-03T09:25:10", 
    "milestone": "", 
    "owner": "olivas", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150311T04:06:40: 
* olivas changed owner from "" to "olivas"
* olivas changed status from "new" to "accepted"

**Comment by icecube-trac** [comment]
Trac update at 20150323T22:42:49: 
* olivas changed component from "cmake" to "icecube offline"
* olivas commented:

> This is also not a cmake issue.

**Comment by icecube-trac** [comment]
Trac update at 20150810T22:25:37: olivas changed milestone from "" to "offline V15-09-00"

**Comment by icecube-trac** [comment]
Trac update at 20151110T17:18:31: david.schultz commented:

> depends on #1420

**Comment by icecube-trac** [comment]
Trac update at 20151113T22:31:59: 
* david.schultz commented:

> In changeset 2345:
> 
> ```text
> #!CommitTicketReference repository="IceTray" revision="2345"
> use the new frame mixer pybinding to save frames. closes #884
> ```


* david.schultz changed resolution from "" to "fixed"
* david.schultz changed status from "accepted" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20160318T21:14:10: 
* olivas changed milestone from "offline V15-12-00" to ""
* olivas commented:

> Milestone offline V15-12-00 deleted

**Comment by icecube-trac** [comment]
Trac update at 20190111T21:50:21: nwhitehorn commented:

> In changeset 169176:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="169176"
> use the new frame mixer pybinding to save frames. closes #884
> ```
